packages/arb-token-bridge-ui/src/components/App/AppConnectionFallbackContainer.tsx

```diff
@@ -1,20 +1,10 @@
-import Image, { ImageProps } from 'next/image'
-import ThreeArbinautsImg from '@/images/three-arbinauts.webp'
-import { ExternalLink } from '../common/ExternalLink'
 import { WalletConnectWarning } from '../common/WalletConnectWarning'
 
 export function AppConnectionFallbackContainer({
```

Comment:
I don't think we show wrong network page anymore? Thinking if we can just remove the entire thing, but maybe in a different PR

Reply:
no, yea i was thinking the same